modules/virt-features-for-storage-matrix.adoc

```diff
@@ -6,32 +6,89 @@
 = {VirtProductName} storage feature matrix
 
 .{VirtProductName} storage feature matrix
+ifdef::virt-features-for-storage[]
 [cols="40%,30%,30%",options="header"]
 |===
-||Virtual machine live migration  |Host-assisted virtual machine disk cloning
+|
+|Virtual machine live migration
+|Host-assisted virtual machine disk cloning
+endif::[]
+ifdef::virt-importing-rhv-vm,virt-importing-vmware-vm[]
+[cols="50%,50%",options="header"]
+|===
+|
+|Virtual machine import
+endif::[]
 
 |OpenShift Container Storage: RBD block-mode volumes
+ifdef::virt-features-for-storage[]
+|Yes
 |Yes
+endif::[]
+ifdef::virt-importing-rhv-vm[]
+ifeval::["{VirtVersion}" == "2.5"]
```

Comment:
Not an issue now but is this going to be updated when we get to 2.6 and beyond?

Reply:
Yes, the bugs have already been created.